spec/models/document_type/title_and_base_path_field_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe DocumentType::TitleAndBasePathField do
+  describe "#payload" do
+    it "returns a hash with title and routing attributes" do
+      edition = build(:edition, title: "Some title", base_path: "/foo/bar/baz")
```

Comment:
For consistency, is it worth making the base path derive from the title? i.e. `/some-title`?

Reply:
I'm struggling a little to see where you're coming from here. Maybe I'm used to test data being pretty random! In this case, there's an argument for keeping it the same as the test it was extracted from. And consistency may be a false friend here: the 'consistent' base path would actually be `/prefix/some-title`.

Reply:
Only suggesting as the title usually drives the base path, so `Some title` would become `some-title`. I don't think it affects the readability of the test particularly - it's not a hill I'd die on! `/prefix/some-title` does seem more accurate than `/foo/bar/baz` though, and gives someone glancing through the codebase a better idea that one affects the other.